okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -41,7 +41,6 @@
 import com.squareup.okhttp.internal.spdy.Http20Draft16;
 import com.squareup.okhttp.internal.spdy.Settings;
 import com.squareup.okhttp.internal.spdy.Variant;
-
```

Comment:
This is the newline that caused this PR when I said should remain present. Were you testing the checkstyle checker and forgot to undo this?